cmake/Jucer2CMake/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2017 Alain Martin
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+cmake_minimum_required(VERSION 3.0)
+
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/../../cmake")
+include(Reprojucer)
+
+
+if(NOT DEFINED Projucer_jucer_FILE)
+  message(FATAL_ERROR "Projucer_jucer_FILE must be defined")
+endif()
+
+
+if(NOT EXISTS "${Projucer_jucer_FILE}")
+  message(FATAL_ERROR "${Projucer_jucer_FILE} must exist")
+endif()
+
+# Explicitely makes Projucer file variable absolute, based the from build directory 
+# (which is the default base directory when calling the "NOT EXISTS" directive 
+# with a relative path).
+# This disambiguates further COPY operations (which are based on the source directory 
+# when called with a relative path).
+get_filename_component(Projucer_jucer_FILE "${Projucer_jucer_FILE}"
+                       REALPATH BASE_DIR "${CMAKE_BINARY_DIR}")
```

Comment:
@MartyLake: please try 
```
get_filename_component(Projucer_jucer_FILE "${Projucer_jucer_FILE}" ABSOLUTE)
```
and I guess it should go before checking for existence? Otherwise I don't understand what it's doing...